spec/views/components/accordion.html.erb_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+include ComponentTestHelper
```

Comment:
This and the method both need to be inside the `describe` block, so they're isolated to this file. Otherwise, the component tests all fail when run together, as the methods override each other.